### PR TITLE
Package 0.5.0rc1

### PR DESCRIPTION
**Pull request by sbesson**
Following the first pre-release of https://github.com/glencoesoftware/bioformats2raw/releases/tag/v0.5.0rc1, this updates the recipes with changes similar to https://github.com/ome/conda-bioformats2raw/pull/5 to build `0.5.0rc1` packages and upload them to the `pre` label of https://anaconda.org/ome/bioformats2raw and https://anaconda.org/ome/bioformats2raw-libs

### CODE REVIEW

**Comment by sbesson** [comment]
After fixing the patch file, this should now be ready for review and deployment to the `ome` channel

File: bioformats2raw/windows-bat-classpath-wildcard.patch [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 +set CLASSPATH=%APP_HOME%\lib\*
  
+ 
  @rem Execute bioformats2raw
- "%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %BIOFORMATS2RAW_OPTS%  -classpath "%CLASSPATH%" com.glencoesoftware.bioformats2raw.Converter %CMD_LINE_ARGS%
```
Comment (joshmoore) [inline bioformats2raw/windows-bat-classpath-wildcard.patch:11]: This is a surprise, no? Or does upstream no longer have this line?

File: .github/workflows/condabuild.yml [hunk 2]
```diff
@@ -82,7 +82,7 @@ jobs:
         run: |
           anaconda whoami
           for f in artifact/bioformats2raw* artifact/*/bioformats2raw*; do
-            anaconda upload --user ${ANACONDA_USER} $f
+            anaconda upload --user ${ANACONDA_USER} $f --label pre
```
Comment (joshmoore) [inline .github/workflows/condabuild.yml]: Makes complete sense, though I assume we could detect the "pre" state of the tag and have an if/else statement here, no?

File: bioformats2raw/windows-bat-classpath-wildcard.patch (hunk 1 above)
Comment (sbesson) [reply bioformats2raw/windows-bat-classpath-wildcard.patch:11]: I think the difference is that the BAT file in 0.5.0-rc1 includes an extra blank line between the CLASSPATH block and the next block so the `diff` is shifted by one row

Comment (sbesson) [reply bioformats2raw/windows-bat-classpath-wildcard.patch:11]: Please don't ask me what caused this in the source build though as I have no idea 😄

File: .github/workflows/condabuild.yml (hunk 2 above)
Comment (sbesson) [reply .github/workflows/condabuild.yml]: Good idea and this should simplify future maintenance - see dbdeb45

**Comment by joshmoore** [comment]
> @sbesson [Automatically deploy rc versions to pre label](https://github.com/ome/conda-bioformats2raw/pull/11/commits/dbdeb451b8e9207824fe0df9e70a7457abaa6d8b)

🤩

**Review by joshmoore — APPROVED** [review]
Nice. I say let's get this deployed and I'll test this morning. If necessary, we'll push another build.

**Comment by sbesson** [comment]
```
(base) sbesson@Sebastiens-MacBook-Pro conda-bioformats2raw % conda search -c ome/label/pre bioformats2raw-libs
Loading channels: done
# Name                       Version           Build  Channel             
bioformats2raw-libs         0.3.0rc3               0  ome/label/pre       
bioformats2raw-libs         0.3.0rc4               0  ome/label/pre       
bioformats2raw-libs         0.5.0rc1               0  ome/label/pre       
```

File: bioformats2raw-libs/meta.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {% set name = "bioformats2raw-libs" %}
-{% set version = "0.4.0" %}
+{% set version = "0.5.0rc1" %}
+{% set github_version = "0.5.0-rc1" %}
```
Comment (jburel) [inline bioformats2raw-libs/meta.yaml:3]: The introduction of the following commit has broken the scheduled version bump

**Pull request merged**